examples/mugen/generation/text_video_gpt.py

```diff
@@ -93,10 +98,18 @@ def text_video_gpt(
     """
 
     # builds text tokenizer from pre-trained
-    text_tokenizer_local_path = _PATH_MANAGER.get_local_path(
-        pretrained_text_tokenizer_url
+    text_tokenizer_encoder_local_path = _PATH_MANAGER.get_local_path(
```

Comment:
Don't think we need to get the local path in this case. TorchText utility should handle it already [here](https://github.com/pytorch/text/blob/5b78d074bd303eb230d30567646fcf0358ee2dd4/torchtext/transforms.py#L471).

Reply:
great catch, thanks!